src/bentoml/_internal/cloud/bentocloud.py

```diff
@@ -68,13 +68,14 @@ def push_bento(
         threads: int = 10,
         context: str | None = None,
     ):
-        with Live(self.spinner.progress_group):
+        with Live(self.spinner.progress_group, transient=True):
             upload_task_id = self.spinner.transmission_progress.add_task(
                 f'Pushing Bento "{bento.tag}"', start=False, visible=False
             )
             self._do_push_bento(
                 bento, upload_task_id, force=force, threads=threads, context=context
             )
+        print(f'✅ Pushed Bento "{bento.tag}"')
```

Comment:
```suggestion
```

Reply:
This is @parano 's suggestion:

<img width="880" alt="image" src="https://github.com/bentoml/BentoML/assets/1206493/b470faed-87d3-44b2-a233-2649e5137070">

Reply:
Then I think instead of print it should be logs?

Reply:
@aarnphm I'm afraid there might be some extra stuff in the logs, like dates or line numbers, which wouldn't be clean. Besides, we're already using rich.Live here, so adding one more print shouldn't matter.

Reply:
Oh I mean console.log